Browsable FAQs/QA/tests/browsable_faqs_test.exs

```diff
@@ -53,6 +53,18 @@ defmodule BrowsableFAQsTest do
       ]
     }
 
+    related_page_leaf = %ContentPage{
+      parent: "topic-1",
+      slug: "related-page-leaf",
+      title: "Related Page Leaf",
+      related_pages: ["multiple-messages-leaf"],
+      wa_messages: [
+        %WAMsg{
+          message: "Test related page leaf"
+        }
+      ]
+    }
+
```

Comment:
There's also a TODO a bit further down about related pages that we can remove.

Reply:
Removed.